RecoTracker/MkFitCMS/standalone/Makefile

```
@@ -20,6 +21,9 @@ all: ${TGTS}
 SRCS := $(wildcard ${CMS_DIR}/src/*.cc) $(wildcard ${SACMS}/*.cc)
 ifdef WITH_ROOT
 SRCS += ${SACMS}/tkNtuple/WriteMemoryFile.cc
+DictsDict.cc ${DICTPCM}: ${SACMS}/tkNtuple/DictsLinkDef.h
+	rootcint -v3 -f $@ $<
+	mv DictsDict_rdict.pcm ${DICTPCM}
```

I'm getting failures here: isn't the line above with `rootcint` already writing to `${DICTPCM}` ?
```suggestion
```

---

If I drop the line with `mv` then the compilation completes OK. However when running I get an error `DictsDict_rdict.pcm is not a ROOT as was expected `.
Then I notice that I also get a `DictsDict_rdict_rdict.pcm` file and if I `mv  DictsDict_rdict_rdict.pcm DictsDict_rdict.pcm`, then the job actually runs.
So, should this actually be 
```suggestion
	mv ../DictsDict_rdict_rdict.pcm ${DICTPCM}
```
? (better parse to a non-hardcoded name though)